iroh-api/benches/get.rs

```diff
@@ -0,0 +1,122 @@
+use cid::Cid;
+use criterion::{black_box, criterion_group, criterion_main, BenchmarkId, Criterion};
+use futures::StreamExt;
+use iroh_api::{Api, OutType, UnixfsConfig, UnixfsEntry};
+use iroh_metrics::config::Config as MetricsConfig;
+use iroh_resolver::resolver::{Path as IpfsPath, Resolver};
+use iroh_rpc_client::{Client, Config as RpcClientConfig};
+use iroh_rpc_types::Addr;
+use iroh_store::{Config as StoreConfig, Store};
+use iroh_unixfs::{
+    chunker::{ChunkerConfig, DEFAULT_CHUNKS_SIZE},
+    content_loader::{FullLoader, FullLoaderConfig},
+};
+use tokio::runtime::Runtime;
+
+fn get_benchmark(c: &mut Criterion) {
+    let mut group = c.benchmark_group("unixfs_get_file");
+    for file_size in [
+        1024,             //  1 KiB
+        1024 * 1024,      //  1 MiB (triggers chunking)
+        10 * 1024 * 1024, // 10 MiB (triggers chunking)
+    ]
+    .iter()
+    {
+        let value = vec![8u8; *file_size];
+        let dir = tempfile::tempdir().unwrap();
+        let path = dir.path().join(format!("{}.raw", file_size));
+        std::fs::write(&path, value).unwrap();
```

Comment:
I guess 10MiB isn't that much to put in memory... Though I'd at least explicitly drop `value` after this.